scalableminds/webknossos
# Update displayed position in statusbar when flycam changes

Currently, the displayed position of the mouse in the statusbar does only update, when the mouse is moved. This PR now also changes the statusbar, so that the position also updates, each time the flycam changes (e.g. via space in z direction).

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a tracing and move the mouse over a segment.
- Now move around with space and the arrow keys.
- The displayed position should always update.

### Issues:
- fixes  #5443

------
